iib/workers/config.py

```diff
@@ -83,13 +83,16 @@ class Config(object):
         'iib.workers.tasks.build_regenerate_bundle',
         'iib.workers.tasks.build_create_empty_index',
         'iib.workers.tasks.build_fbc_operations',
+        'iib.workers.tasks.build_add_deprecations',
         'iib.workers.tasks.general',
     ]
     # Path to hidden location of SQLite database
     hidden_index_db_path: str = '/var/lib/iib/_hidden/do.not.edit.db'
     # path where catalog resides in fbc_fragment
     # might need to be changed, currently based on test fbc-fragment
     fbc_fragment_catalog_path: str = '/configs'
+    # path where operator deprecations will be stored
+    operator_deprecations_dir: str = 'operator-deprecations-content'
```

Comment:
Could you please add this to README?

Reply:
And is this the default path correct? 
Right now it defaults to: `/config/operator-deprecations-content`

Reply:
yup, operator_deprecations_dir is just the name of dir, the path will default to /config/operator-deprecations-content/

Reply:
@chandwanitulsi I still have some questions about this directory - should not this be set by full path (similar to variables above)? 
Is there any agreement with other teams that the path for deprecation must be in /configs folder? 
Would not be better to have this set the path as image LABEL as well?

Reply:
as we agreed on standup - lets update comment that this folder is subfolder of `fbc_fragment_catalog_path`
and maybe consider changing the name to start with something else then letter. For example "_" 
`_operator-deprecations-content`